docs/physics/spectrum/basic.ipynb

```diff
@@ -0,0 +1,387 @@
+{
```

Comment:
<p>"This simply requires knowledge of the each packet's energy and frequency" --&gt; "This simply requires knowledge of each packet's energy and frequency"</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/1744/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>